ubcpi/static/html/ubcpi_edit.html

```diff
@@ -78,7 +78,7 @@
                             <div class="wrapper-comp-setting ubcpi-options-list" id="option-{{ $index + 1}}" ng-repeat="option in esc.data.options track by $index">
                                 <div class="pi-label-and-hint pi-repeater">
                                     <label class="label setting-label pi-setting-label" for="pi-option-{{$index}}" aria-describedby="pi-option={{$index}}-tip" translate>Answer {{$index + 1}}</label>
-                                    <span id="pi-option={{$index}}-tip" class="tip setting-help" translate>Add the text for this answer. If you wish to add an image, press the button below.</span> <input type="button" class="button action-primary add-image-button" value="{{esc.data.options[$index].image_show_fields ? 'Remove Image' | translate : 'Add Image' | translate }}" ng-click="esc.image_show_fields($index)" name="add-image-button-pi-option-{{$index+1}}" />
+                                    <span id="pi-option={{$index}}-tip" class="tip setting-help" translate>Add the text for this answer. If you wish to add an image, press the button below.</span> <input type="button" class="button action-primary add-image-button" value="{{ (esc.data.options[$index].image_show_fields ? 'Remove Image' : 'Add Image') | translate }}" ng-click="esc.image_show_fields($index)" name="add-image-button-pi-option-{{$index+1}}" />
```

Comment:
When you run the extract, do both strings get extracted? If not, we should be something in place to prevent the loss of the translation strings